docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -399,6 +399,29 @@ To deploy Elastic Agent in clusters with the Pod Security Policy admission contr
 
 By default, ECK creates a Service for Fleet Server that Elastic Agents can connect through. You can customize it using the `http` configuration element. You can read more about link:k8s-services.html[making changes] to the Service and link:k8s-tls-certificates.html[customizing] TLS configuration in the documentation.
 
+[id="{p}-elastic-agent-fleet-configuration-override-default-fleet-configuration-settings"]
+=== Override default Fleet configuration settings
+
+ECK uses environmental variables to control how Elastic Agent and Fleet Server should be configured. Sometimes, it might be required to override some of those settings. For example, if Kibana TLS certificate is signed by a well-known root and can't include `kibana-kb-http.namespace.svc` as a SAN, `KIBANA_FLEET_HOST` can be overriden to point to the URL that the certificate specifies. To do that, specify environmental variable, as below.
```

Comment:
```suggestion
ECK uses environmental variables to control how Elastic Agent and Fleet Server should be configured. Sometimes, it might be required to override some of these settings. For example, if Kibana TLS certificate is signed by a well-known root and can't include `kibana-kb-http.namespace.svc` as a SAN, `KIBANA_FLEET_HOST` can be overriden to point to the URL that the certificate specifies. To do that, specify environmental variable, as shown in the following example.
```

Reply:
```suggestion
ECK uses environment variables to control how Elastic Agent and Fleet Server should be configured. Sometimes, it might be required to override some of these settings. For example, if Kibana TLS certificate is signed by a well-known root and can't include `kibana-kb-http.namespace.svc` as a SAN, `KIBANA_FLEET_HOST` can be overriden to point to the URL that the certificate specifies. To do that, specify environmental variable, as shown in the following example.
```